src/init/AclInitializer.ts

```diff
@@ -49,22 +50,8 @@ export class AclInitializer extends Initializer {
     }
   }
 
-  // Set up ACL so everything can still be done by default
-  // Note that this will need to be adapted to go through all the correct channels later on
   protected async setRootAclDocument(rootAcl: ResourceIdentifier): Promise<void> {
-    const acl = `@prefix   acl:  <http://www.w3.org/ns/auth/acl#>.
-@prefix  foaf:  <http://xmlns.com/foaf/0.1/>.
-
-<#authorization>
-    a               acl:Authorization;
-    acl:agentClass  foaf:Agent;
-    acl:mode        acl:Read;
-    acl:mode        acl:Write;
-    acl:mode        acl:Append;
-    acl:mode        acl:Delete;
-    acl:mode        acl:Control;
-    acl:accessTo    <${this.baseUrl}>;
-    acl:default     <${this.baseUrl}>.`;
+    const acl = await fsPromises.readFile(joinFilePath(__dirname, '../../templates/root/.acl'), 'utf8');
```

Comment:
I would have the class take an (optional) input parameter defining the path of the root acl file (with fallback to this one if no input is given). That way in the future custom files can be used for the root without having to change the package.